src/declaration.d

```diff
@@ -653,19 +653,22 @@ extern (C++) final class AliasDeclaration : Declaration
             type = null;
             aliassym = s;
         }
+        inuse = 0;
         if (global.gag && errors != global.errors)
         {
             type = oldtype;
             aliassym = null;
         }
-        inuse = 0;
-        semanticRun = PASSsemanticdone;
-
-        if (auto sx = overnext)
+        else
         {
-            overnext = null;
-            if (!overloadInsert(sx))
-                ScopeDsymbol.multiplyDefined(Loc(), sx, this);
+            semanticRun = PASSsemanticdone;
```

Comment:
Moving `semanticRun` here means repeated semantic for the error case. Is this intended?